### PR TITLE
[platform] http client timeout

### PR DESCRIPTION
**Pull request by milchakov**

### CODE REVIEW

File: storage/diff_scheme/diff_scheme_checker.cpp [hunk 1]
```diff
@@ -25,6 +25,8 @@ char const kNameKey[] = "name";
 char const kSizeKey[] = "size";
 char const kVersionKey[] = "version";
 
+auto const kTimeoutSeconds = 5.0;
```
Comment (VladiMihaylenko) [inline storage/diff_scheme/diff_scheme_checker.cpp]: s/kTimeoutInSeconds/kTimeoutInSeconds

Comment (milchakov) [inline storage/diff_scheme/diff_scheme_checker.cpp]: готово

File: platform/http_client.hpp [hunk 2]
```diff
@@ -77,6 +77,7 @@ class HttpClient
   // TODO: "false" is now supported on Android only.
   HttpClient & SetHandleRedirects(bool handle_redirects);
   HttpClient & SetRawHeader(string const & key, string const & value);
+  void SetTimeout(double seconds);
```
Comment (ygorshenin) [inline platform/http_client.hpp]: s/seconds/timeoutSec/

Comment (milchakov) [inline platform/http_client.hpp]: готово

**Comment by milchakov** [comment]
@ygorshenin PTAL

**Pull request merged**